src/picongpu/include/fields/numericalCellTypes/EMFCenteredCell.hpp

```diff
@@ -129,7 +129,8 @@ namespace traits
     template<uint32_t T_simDim>
     struct FieldPosition<FieldTmp, T_simDim>
```

Comment:
pls update doxygen string:
```C++
/** position (floatD_X in case of T_simDim == simDim) in cell, wrapped in
 * one-component vector since it's a scalar field with only one component, for the
 * scalar field FieldTmp
 */
```